packages/sit-onyx/src/components/OnyxSwitch/types.ts

```diff
@@ -0,0 +1,31 @@
+export type OnyxSwitchProps = {
+  /**
+   * Whether the switch should be checked or not.
+   */
+  modelValue?: boolean;
+  /**
+   * The label of the switch.
+   */
+  label?: string;
+  /**
+   * Whether to disable the switch and prevent user interaction.
+   */
+  disabled?: boolean;
+  /**
+   * Whether the switch should be readonly or not.
+   */
+  readonly?: boolean;
+  /**
+   * Whether the switch is required / has to be checked.
+   */
+  required?: boolean;
+  /**
+   * Whether the switch should be invalid or not.
+   */
+  invalid?: boolean;
```

Comment:
Please name the prop `errorMessage` instead.
```suggestion
  errorMessage?: string;
```